com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java

```diff
@@ -472,23 +494,41 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException,
 	 * 
 	 * @return row data updated
 	 */
+	public String[] updateTableImpData(String owner, String tableName, long imp) {
+		
+		for (String[] item : tableItems) {
```

Comment:
If only check item[0] and item[5] , I don't think i  need "for loop".

Reply:
* I have the same opinion on this as I wrote above.